cmd/helm/repo_add.go

```diff
@@ -186,6 +186,8 @@ func (o *repoAddOptions) run(out io.Writer) error {
 
 	f.Update(&c)
 
+	f.Generated = time.Now()
```

Comment:
This introduces potentially dangerous behaviour. This timestamp is intended to demonstrate when the index file was originally generated by `helm repo index` on the server-side, not when `helm repo add` was called. Instead this change will mask that generation timestamp.

I don't think we can accept this change.